install/installer/traf_config_check

```diff
@@ -545,7 +545,7 @@ else
 
    hadoopVersion=$(curl -k -su $ADMIN:$PASSWORD $URL/api/v1/clusters | grep version | grep -c CDH)
 
-   if [[ $hadoopVersion -ne "1" ]]; then
+   if [[ $hadoopVersion -lt "1" ]]; then
       hadoopVersion=$(curl -k -su $ADMIN:$PASSWORD $URL/api/v1/clusters | grep version | grep -c HDP)
```

Comment:
That a good question. I only tested on a multi-cluster setup with 2 clusters. I think it would, but I will test this in the future.